.github/renovate.json

```diff
@@ -126,7 +126,7 @@
         "^latest\\/base.yml$"
       ],
       "matchStrings":[
-        "# renovate: datasource=(?<datasource>github-releases) depName=(?<depName>.*?)\n.*?docker: '5:(?<currentValue>.*?)'"
+        "# renovate: datasource=(?<datasource>github-releases) depName=(?<depName>.*?)\n.*?docker: '5:^v(?<currentValue>.*?)'"
```

Comment:
Are you sure about the "^"? Usually this should only be used as anchor at the beginning of a regex.

Reply:
Alternatively, to strip a release- prefix:


{
  "packageRules": [
    {
      "matchPackageNames": ["bar"],
      "extractVersion": "^release-(?<version>.*)$"
    }
  ]
}
The above will change a raw version of release-2.0.0 to 2.0.0, for example. A similar one could strip leading v prefixes:


{
  "packageRules": [
    {
      "matchPackageNames": ["baz"],
      "extractVersion": "^v(?<version>.*)$"
    }
  ]
}

This is an example from the github-renovate-documentation.

https://docs.renovatebot.com/configuration-options/#extractversion

Reply:
exactly, the "^" is used as anchor at the beginning of the regex to match the beginning of the line, just like the "$" is used as anchor attached to the end of the line. neither should be used in the middle of the regex. so you should drop it from your code here

Reply:
done